package.json

```diff
@@ -111,6 +113,9 @@
       "plugins:preinstall": [
         "./src/hooks/plugin-install"
       ],
+      "prerun": [
```

Comment:
Adding a new prerun hook to warn users about the update feature not working for npm based installations: https://github.com/oclif/plugin-update/issues/49